boa/src/builtins/array/mod.rs

```diff
@@ -276,6 +276,29 @@ impl Array {
         Ok(array_obj_ptr)
     }
 
+    /// Utility function for concatenating array objects.
+    ///
+    /// Returns a Boolean valued property that if `true` indicates that
+    /// an object should be flattened to its array elements
+    /// by `Array.prototype.concat`.
+    pub(crate) fn is_concat_spreadable(this: &Value, context: &mut Context) -> Result<bool> {
+        if !this.is_object() {
```

Comment:
```suggestion
		// 1. If Type(O) is not Object, return false.
        if !this.is_object() {
```
It would be nice to document according to spec.